xmppserver/src/main/webapp/log.jsp

```diff
@@ -109,36 +109,17 @@
     try {
         String line;
         int totalNumLines = 0;
-        BufferedReader in = null;
-        InputStreamReader inputStreamReader= null;
-        FileInputStream fileInputStream=null;
-        try{
-        	fileInputStream = new FileInputStream(logFile);
-        	inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8");
-        	in = new BufferedReader(inputStreamReader);
+        try(FileInputStream fileInputStream = new FileInputStream(logFile); InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8"); BufferedReader in = new BufferedReader(inputStreamReader);){
```

Comment:
Ugh. Mind adding some newlines? Something like this is perfectly valid.
```
try( FileInputStream fileInputStream = new FileInputStream(logFile);
     InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8");
     BufferedReader in = new BufferedReader(inputStreamReader); )
{
```

Reply:
My mistake! I disabled automatic code formatting to keep whitespace changes at a minimum.

Reply:
done.
Should I make a new pull request with only one commit?

Reply:
Don't bother, this is fine.